### ISSUE TITLE
`import dtale` changes matplotlib settings

### ISSUE THREAD

**Issue by BlackHC** [man-group/dtale#703]
DTale is great. However, when I import it, it changes my matplotlib settings.

See https://cs.github.com/man-group/dtale/blob/3e8d1ce16406a84a9c7293589aa4a8f9b17fb78f/dtale/views.py#L25

```
import matplotlib

matplotlib.use("agg")  # noqa: E261

import matplotlib.pyplot as plt

plt.rcParams["font.sans-serif"] = ["SimHei"]  # Or any other Chinese characters
matplotlib.rcParams["font.family"] = ["Heiti TC"]
```

This is not great when using jupyter lab as suddently plots don't display correctly anymore (and I don't have the Heiti TC font, resulting in lots of warnings).

**Comment by aschonfeld** [comment]
@BlackHC apologies for the issues.  I'm working on trying to add some warning filtering and moving the import to within the function that uses it so that unless you use the function (exporting Correlation matrix images) then it shouldn't affect your jupyter sessions

**Comment by aschonfeld** [comment]
@BlackHC just released v2.9.0 with this fix.  Please let me know how it goes.  Also, if you haven't already, please toss your ⭐  on the repo when you get a chance.  Thanks 🙏
